lib/Ama/Model/OAuth2.pm

```diff
@@ -25,7 +25,6 @@ sub store {
       $self->pg->db->query('insert into providers (id, provider_id, provider) values (?, ?, ?)', $id, $mapped->{id}, $provider);
     }
   } else {
```

Comment:
ah. There was a warn statement within that else block. I removed all warn statements